e2e/controller_test.go

```diff
@@ -0,0 +1,72 @@
+package e2e
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+)
+
+func testController() {
+	var cc CleanupContext
+	BeforeEach(func() {
+		cc = commonBeforeEach()
+	})
+	AfterEach(func() {
+		commonAfterEach(cc)
+	})
+
+	It("should be deployed", func() {
+		Eventually(func() error {
```

Comment:
Can't you replace this `Eventually` section with `kubectl wait` command?

For example:
```
kubectl("get", "-n=topolvm-system", "deploy", "--selector=app.kubernetes.io/component=controller,app.kubernetes.io/name=topolvm", "--for=condition=Available", "--timeout=120s"
```
(Please change the timeout value to something good.)

Reply:
I think that is correct.

Reply:
I fixed it. Please review.